master/usage/calicoctl/configure/etcd.md

```diff
@@ -5,15 +5,15 @@ canonical_url: 'https://docs.projectcalico.org/v3.1/usage/calicoctl/configure/et
 
 ## Complete list of etcdv3 configuration options
 
-| Configuration file option  | Environment variable | Description                                                                           | Schema
-| ---------------------------| -------------------- | ------------------------------------------------------------------------------------- | ------
-| `datastoreType`            | `DATASTORE_TYPE`     | Indicates the datastore to use. If unspecified, defaults to `etcdv3`. (optional)      | `kubernetes`, `etcdv3`
-| `etcdEndpoints`            | `ETCD_ENDPOINTS`     | A comma separated list of etcd endpoints. Example: `http://127.0.0.1:2379` (required) | string
-| `etcdUsername`             | `ETCD_USERNAME`      | User name for RBAC. Example: `user` (optional)                                        | string
-| `etcdPassword`             | `ETCD_PASSWORD`      | Password for the given user name. Example: `password` (optional)                      | string
-| `etcdKeyFile`              | `ETCD_KEY_FILE`      | Path to the etcd key file. Example: `/etc/calico/key.pem` (optional)                  | string
-| `etcdCertFile`             | `ETCD_CERT_FILE`     | Path to the etcd client certificate, Example: `/etc/calico/cert.pem` (optional)       | string
-| `etcdCACertFile`           | `ETCD_CA_CERT_FILE`  | Path to the etcd Certificate Authority file. Example: `/etc/calico/ca.pem` (optional) | string
+| Configuration file option | Environment variable | Description                                                                           | Schema
+| --------------------------| -------------------- | ------------------------------------------------------------------------------------- | ------
+| `datastoreType`           | `DATASTORE_TYPE`     | Indicates the datastore to use. If unspecified, defaults to `etcdv3`. (optional)      | `kubernetes`, `etcdv3`
+| `etcdEndpoints`           | `ETCD_ENDPOINTS`     | A comma separated list of etcd endpoints. Example: `http://127.0.0.1:2379` (required) | string
+| `etcdUsername`            | `ETCD_USERNAME`      | User name for RBAC. Example: `user` (optional)                                        | string
+| `etcdPassword`            | `ETCD_PASSWORD`      | Password for the given user name. Example: `password` (optional)                      | string
+| `etcdKeyFile`             | `ETCD_KEY_FILE`      | Path to the file containing the private key of the client certificate issued to `calicoctl` by the etcd certificate authority (CA). Enables `calicoctl` to participate in mutual TLS authentication and identify itself to the etcd server. Example: `/etc/calico/key.pem` (optional) | string
```

Comment:
I was a little confused by the phrase "etcd certificate authority" when I first read it; thinking, etcd isn't a certificate authority!

What we really mean is "the CA that issued the etcd server certificate(s)" but that is a bit of a mouthful. Your call.

Reply:
Well, actually, that's not even right, since calicoctl and the etcd server could technically use different CAs for their certs.

Actually, we should drop the phrase "by the etcd certificate authority (CA)". Saying the cert was issued by a CA is redundant, and saying it's the "etcd" CA just confuses things.

Reply:
I am not sure they really can use different CAs. It seems that there is a "trusted CA" for etcd and that the client certs must be signed by that CA. https://coreos.com/etcd/docs/latest/op-guide/security.html

Reply:
the "trusted CA" is the CA that the server will trust for client certs. It doesn't say that the server cert (`--cert-file`) needs to be signed by that same CA.

Reply:
OK I guess you're right 😄